src/dolphin/workflows/config/_common.py

```diff
@@ -204,6 +210,13 @@ class UnwrapOptions(BaseModel, extra="forbid"):
         description="Statistical cost mode method for SNAPHU.",
     )
 
+    alpha: float = Field(
```

Comment:
we might end up changing this name slightly to avoid confusion with [the SHP alpha parameter](https://github.com/isce-framework/dolphin/blob/deb239e4a75bf41b0ced162156726d3fd127ddef/src/dolphin/workflows/config/_common.py#L101) but for now it's fine and different